eng/mgmt/mgmtmetadata/compute_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=D:\Swagger\azure-sdk-for-net\sdk
-2020-04-23 20:17:25 UTC
+cmd.exe /c autorest.cmd https://github.com/mirichmo/azure-rest-api-specs/blob/mirichmo-assess-patches/specification/compute/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=F:\GitRepos\azure-sdk-for-net\sdk
```

Comment:
We do not allow SDK to be generated from any fork/branch other than Azure/master. Please generate by executing `src/generate.ps1`